vignettes/col_plan.Rmd

```diff
@@ -192,6 +211,33 @@ tfrmt(
   print_to_gt(dat)
 ```
 
+Renaming column spanners can also be specified using `span_structure`. We can rename multiple spanners _within the same level_ in the same `span_structure` - here `"cols 1,2"` becomes `"first cols"` and `"col 4"` becomes `"just col4"` for the `span1` level. But when renaming multiple spanners _in different levels_ we need to create a new `span_structure` for each level - so we write `"column cols"` becomes `"most columns"` for the `span2` level in a separate `span_structure`.
+
+```{r}
+tfrmt(
+  group = group,
+  label = label,
+  param = parm,
+  value = val,
+  column = c(span2, span1, my_col),
+  body_plan = body_plan(
+    frmt_structure(group_val = ".default", label_val = ".default", frmt("x"))
+  ),
+  col_plan = col_plan(
+    # rename column spanner in same level
+    span_structure(span1 = c("first cols" = "cols 1,2"),
+                   span1 = c("just col4" = "col 4")),
```

Comment:
these should be combined into a single named vector:
````
span_structure(span1 = c("first cols" = "cols 1,2",
                                          "just col4" = "col 4"))
```

Reply:
addressed here: https://github.com/GSK-Biostatistics/tfrmt/pull/420/commits/0472a0c8453ced9154a8523557af69513b7822fe

please resolve conversation if new commit meets requirements :)